docs/admin/authentication.md

```diff
@@ -470,4 +470,8 @@ Finally, add the following parameters into API server start parameters:
 
           openssl x509  -noout -text -in ./server.crt
 
-Finally, do not forget to fill out and add the same parameters into the API server start parameters.
+1.  Fill in and add the following parameters into the API server start parameters:
+
+          --client-ca-file=/yourdirectory/ca.crt
+          --tls-cert-file=/yourdirectory/server.crt
+          --tls-private-key-file=/yourdirectory/server.key
```

Comment:
Can you add `--tls-ca-file` too?  It's used in 1.5 to allow the API server to make loopback connections to itself on the secure API port.